Integrations/MimecastV2/MimecastV2.py

```diff
@@ -460,7 +481,7 @@ def create_policy():
         'Reciever': {
             'Group': reciever.get('groupId'),
             'Email Address': reciever.get('emailAddress'),
-            'Domain': reciever.get('domain'),
+            'Domain': reciever.get('emailDomain'),
```

Comment:
why did we make this change?

Reply:
Cause It isnt the right name, the right name is emailDomain